Makefile

```
@@ -174,3 +178,51 @@ watch-ts: ## watch changes in TypeScript files
 help:
 	@grep -E '^[a-zA-Z0-9_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 .PHONY: help
+
+###########################################
+# Translations tasks
+
+.PHONY: i18n-generate
+i18n-generate: ## Generate source translations files for all applications
+	${MAKE} i18n-generate-back;
+	${MAKE} i18n-generate-front;
+
+.PHONY: i18n-compile
+i18n-compile: ## Compile translated messages to be used by all applications
+	${MAKE} i18n-compile-back
+	${MAKE} i18n-compile-front
+
+.PHONY: i18n-generate-front
+i18n-generate-front:
+	@$(YARN) build
+	@$(YARN) generate-l10n-template
+
+.PHONY: i18n-compile-front
+i18n-compile-front:
+	@$(YARN) generate-translations
+
+.PHONY: i18n-generate-back
+i18n-generate-back:
+	@$(COMPOSE_RUN) -w /app/src/richie app python /app/sandbox/manage.py makemessages --ignore "venv/**/*" --keep-pot
+
+.PHONY: i18n-compile-back
+i18n-compile-back:
+	@$(COMPOSE_RUN) -w /app/src/richie app python /app/sandbox/manage.py compilemessages
+
+.PHONY: crowdin-upload
+crowdin-upload: ## Upload source translations to crowdin
```

```suggestion
crowdin-upload: ## Upload source translations to Crowdin
```

Same in other targets?